triemux/mux.go

```diff
@@ -37,6 +40,13 @@ func NewMux() *Mux {
 // If the routing table is empty, return a 503.
 func (mux *Mux) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if mux.count == 0 {
+		defer func() {
+			logger.NotifySentry(logger.ReportableError{
+				Error: logger.RecoveredError{"Route table is empty!"},
+				Request: r,
+			})
+			InternalServiceUnavailableCountMetric.With(prometheus.Labels{"host": r.Host}).Inc()
```

Comment:
Is `r.Host` always going to be `www.gov.uk`? Is it worth having that label?

Reply:
True, I'll remove it. The host label is probably not useful - I added it since other count metrics have it.

The label for `TEMPORARY_CHILD` should work. I'm concerned that Router seems to be briefly [unreachable during deploys](https://prometheus.publishing.service.gov.uk/graph?g0.expr=up%7Binstance%3D%2210.13.5.208%3A3055%22%7D&g0.tab=0&g0.stacked=0&g0.range_input=20m&g0.end_input=2021-03-16%2017%3A20%3A13&g0.moment_input=2021-03-16%2017%3A20%3A13); I am not sure if we actually scrape the temporary child.